pkg/registry/installer.go

```diff
@@ -50,7 +50,8 @@ type Installer interface {
 func NewInstaller(currentDeployHost []net.IP,
 	regConfig *v2.LocalRegistry,
 	infraDriver infradriver.InfraDriver,
-	distributor imagedistributor.Distributor) Installer {
+	distributor imagedistributor.Distributor,
+) Installer {
```

Comment:
ditto

BTW, in general, before the code is reviewed by the reviewers, we should develop a good habit, that is, review our own code firstly, and try to ensure that there are no unnecessary changes. Of course, good modifications are welcome